packages/salesforcedx-vscode-lwc/snippets/lwc-js.json

```diff
@@ -260,10 +260,10 @@
       "",
       "@wire(graphql, {",
       "  query: gql`",
-      "    query ${1:queryName}($recordId: Id) {",
+      "    query ${1:queryName}(\\$recordId: Id) {",
```

Comment:
The `$` in the three spots in this snippit is intended to remain so it has to be escaped.